src/js/blocks/panel/block.js

```diff
@@ -0,0 +1,298 @@
+						<div class={classes}>
+							<div class="cl-panel-super-blur"></div>
+							<div class="cl-panel-super-content">
+								<div class="cl-panel-super-image">
+									<MediaUpload
+										onSelect={ media =>
+											{
+												setAttributes({
+													alt: media.alt,
+													img: media.url,
+													mediaID: media.id
+												});
+											}
+										}
+										type="image"
+										value={ attributes.mediaID }
+										render={ ({ open }) => getImageButton( open ) }
+									/>
+								</div>
+								<div class="cl-panel-super-text">
+									<InnerBlocks
+										allowedBlocks={ ALLOWED_BLOCKS }
+										template={TEMPLATE}
+									/>
+								</div>
+							</div>
+						</div>
+					</div>
+				);
+			} else {
+				return (
+					<div className="container">
+						<div class={classes}>
+							<figure class="poster">
+								<MediaUpload
+									onSelect={ media =>
+										{
+											setAttributes({
+												alt: media.alt,
+												img: media.url,
+												mediaID: media.id
+											});
+										}
+									}
+									type="image"
+									value={ attributes.mediaID }
+									render={ ({ open }) => getImageButton( open ) }
+								/>
+							</figure>
+							<article>
+								<InnerBlocks
+									allowedBlocks={ ALLOWED_BLOCKS }
+									template={TEMPLATE}
+								/>
+							</article>
+						</div>
+					</div>
+				);
+			}
+		};
+
+		const createBlockControls = () => {
+			return (
+				<BlockControls key="controls">
+					{ !! attributes.img && (
+					<MediaUploadCheck>
+						<Toolbar>
+							<MediaUpload
+								onSelect={ media =>
+									{
+										setAttributes({
+											alt: media.alt,
+											img: media.url,
+											mediaID: media.id
+										});
+									}
+								}
+								allowedTypes={ ALLOWED_MEDIA_TYPES }
+								value={ attributes.mediaID }
+								render={ ( { open } ) => (
+									<IconButton
+										className="components-toolbar__control"
+										label={ __( 'Edit media' ) }
+										icon="edit"
+										onClick={ open }
+									/>
+								) }
+							/>
+						</Toolbar>
+					</MediaUploadCheck>
+					) }
+
+				</BlockControls>
+			);
+		};
+
+		// Generate sidebar inspector controls for other custom attributes
+		const createInspectorControls = () => {
+			return (
+				<InspectorControls>
+					<PanelBody>
+					
+						<PanelRow>
+							<BaseControl
+								label={ __( 'Format' ) }
+								help={ __( 'To increase performance, super panel previews will appear simplified in the editor window.' )}
+							>
+								<ButtonGroup aria-label={ __( 'Panel Format' ) }>
+									{ [ 'default', 'super' ].map( ( value ) => {
+										const capitalizedValue = value.charAt( 0 ).toUpperCase() + value.slice( 1 );
+										const key = ( 'default' === value ) ? '' : value;
+										const format = ( undefined == attributes.format ) ? '' : attributes.format;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='. (eqeqeq)](https://app.codacy.com/manual/uriweb/uri-component-library/pullRequest?prid=4757242)